### PR TITLE
Object level retention tests for Compliance and Governance CEPH-83574058

### PR DESCRIPTION
**Pull request by TejasC88**
T2 tests for object locking using object level policy using 
- Compliance mode
- Governance mode

Pass log:
http://magna002.ceph.redhat.com/ceph-qe-logs/tejas/cephci-run-XA4CC2/

### CODE REVIEW

**Review by viduship — APPROVED** [review]

File: suites/pacific/rgw/tier-2_rgw_regression.yaml [hunk 1]
```diff
@@ -100,36 +100,36 @@ tests:
...
-                - service_type: alertmanager
-                  placement:
-                    count: 1
-                - service_type: node-exporter
-                  placement:
-                    host_pattern: "*"
-                - service_type: crash
-                  placement:
-                    host_pattern: "*"
+  #- test:
```
Comment (ckulal) [inline suites/pacific/rgw/tier-2_rgw_regression.yaml:103]: Any reason why monitoring service deployment test-case is commented?
please remove the comment if it is not intended.

**Review by ckulal — COMMENTED** [review]
Also please add Compliance and Governance test-case in quincy suite,
https://github.com/red-hat-storage/cephci/blob/master/suites/quincy/rgw/tier-2_rgw_regression.yaml

**Comment by TejasC88** [comment]
> Also please add Compliance and Governance test-case in quincy suite,
> https://github.com/red-hat-storage/cephci/blob/master/suites/quincy/rgw/tier-2_rgw_regression.yaml

thanks @ckulal , updated , pls review

**Review by ckulal — COMMENTED** [review]
@Tejas, small clarification
don't we need this test case in pacific suite as well?

**Review by anrao19 — APPROVED** [review]

**Review by ckulal — APPROVED** [review]

**Pull request merged**